src/EFCore/ModelBuilder.cs

```diff
@@ -250,6 +338,23 @@ public virtual ModelBuilder Ignore([NotNull] Type type)
             return this;
         }
 
+        /// <summary>
+        ///     Excludes the given entity type from the model. This method is typically used to remove types from
+        ///     the model that were added by convention.
+        /// </summary>
+        /// <param name="name"> The name of the entity type to be removed from the model. </param>
+        /// <returns>
+        ///     The same <see cref="ModelBuilder" /> instance so that additional configuration calls can be chained.
```

Comment:
```suggestion
        ///     The same <see cref="ModelBuilder" /> instance so that additional configuration calls can be chained, or <see langword="null" /> if the entity type does not exist.
```
(Assuming that is correct.)

Reply:
No, Fluent API never return `null`

Reply:
OK. Then I would just say somewhere that there is no problem if you `Ignore` something that doesn't exist.